src/mumble/MainWindow.h

```diff
@@ -269,6 +269,7 @@ class MainWindow : public QMainWindow, public MessageHandler, public Ui::MainWin
 		void pttReleased();
 		void whisperReleased(QVariant scdata);
 		void onResetAudio();
+		void onChangeMute();
```

Comment:
A short Doxygen-comment would be nice for this function as well :)

Reply:
2d30973d1b5324dd4255bb54b34d6902ae7bc859.